# scrap_sandwich_id: [a,b,c,d,e,f,g,h]
scrap_sandwich_id: [a,g,d,h]
# scrap_sandwich_id: []
